examples/02-plot/gif.py

```diff
@@ -21,6 +21,8 @@
 plotter = pv.Plotter()
 plotter.add_mesh(grid, scalars=z.ravel())
 
+print('Orient the view, then press "q" to close window and produce movie')
```

Comment:
I would say `close the window and produce the movie`

Reply:
The "q" part is necessary though. "Close" could be interpreted as pressing the exit button which will destroy the GL context and result in the error in #627.

I'm okay with it as is